### ISSUE TITLE
DSKBYTR WORDEQUAL can be set before DATA visible

### ISSUE THREAD

**Issue by mras0** [tonioni/WinUAE#249]
Minor detail, and I doubt any software depends on it, but I don't think this is currently implemented in WinUAE.

The data visible in the DSKBYTR register seems to be delayed compared to the value used for WORDSYNC. Here's data captured on an A1000 (with real floppy drive):

![image](https://user-images.githubusercontent.com/9952017/209967614-1c29a664-7789-4d99-9b03-c4f61a7c46b4.png)

The highlighted sequence shows that $5044 and $D089 was read from DSKBYTR (i.e. WORDEQUAL was set before the $89 byte was ready).

I know you prefer smaller test cases, so I made a simpler one that reads a track 41 times and counts how often $5044 is read from DSKBYTR. On my A1200 w/ gotek it's ~70-80 times (the count is displayed in hex), while in WinUAE it seems to never happen.

[dskbytr.zip](https://github.com/tonioni/WinUAE/files/10319953/dskbytr.zip) (apologies in advance for the stupid code)

**Comment by tonioni** [comment]
Yeah, there is something unexplained and my plan was always been to examine disk details before 5.0.

I thought DSKBYTR 8-bits of data are directly from internal buffer that is also used for DSKSYNC comparison but that isn't true. 

Easy test: load something, then deselect drives, check all possible DSKSYNC values and exit loop if WORDSYNC bit gets set after writing to DSKSYNC. Value can be totally different than DSKBYTR 8-bit data.
